### DISCUSSION TITLE
Kick by the bad packets

### DISCUSSION THREAD

**Discussion by ytalexyt033** [ORelio/Minecraft-Console-Client#2235] [Q&A]
I just log on to the server and get a kick for bad packets.

**Comment by BruceChenQAQ** [comment]
Try the [latest release](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest).

**Reply by ytalexyt033** [reply]
yes, i use it. Ip mc.holyworld.ru 
if i join any server on it = kick

**Reply by ytalexyt033** [reply]
with out bot work correctly server

**Reply by BruceChenQAQ** [reply]
A new version was just released a few minutes ago, is that what you are using?

**Reply by ytalexyt033** [reply]
yes

**Reply by BruceChenQAQ** [reply]
Try to use the latest build, that fixes the problem of being detected as cheating in holyworld.

**Reply by BruceChenQAQ** [reply]
And change these settings:
```toml
MinecraftVersion = "1.16.5"
BrandInfo = "vanilla"
TerrainAndMovements = true
```
